New blog post - February EcoDev Roundup (#29)
waku-org/blog.waku.org: posts/2024-03-01-february-EcoDev-Roundup.mdx

* Waku latency simulations have been added to a real environment. [Check out the pull request ](https://github.com/waku-org/research/pull/85)and [read the docs](https://docs.waku.org/research/research-and-studies/message-propagation/) on message propagation times with Waku-RLN.
* Waku Relay is responsible for disseminating messages but cannot guarantee delivery. The push-based propagation (PubSub) of Relay can be augmented with a pull-based method, in our case, Sync. [Check out the Waku sync overview first draft](https://github.com/waku-org/research/issues/80).

---

## **Waku Community Call and AMA**

Waku hosts a Community Call on X during the last week of each month. The objective of the call is to update the community on progress and provide a space where questions can be put directly to the team.

The February Community Call took place last week on the 28th of January. You can [listen back to the full Space](https://twitter.com/i/spaces/1lDxLPQrqgoxm/peek), and read the highlights below.

[Guru](https://twitter.com/hackyguru), and [Danish](https://twitter.com/danisharora099) from the Waku team led the call.

### nwaku node campaign

> Guru announced several initiatives for Waku node runners. Anyone who is running an nwaku node is invited to post a photo on X, tagging @waku_org and using #ProofOfNode. Waky will share your post and give you a shoutout. He also announced that they were working on early-stage node operator and onboarding programs that will be formally announced in the coming weeks. Currently, the Wak team is collecting feedback from operators on their experience setting up a node; [drop a message on Discord](https://discord.com/invite/gMPAzmcDER) to leave any feedback you may have. 

Reviewer: ```suggestion
Guru announced several initiatives for Waku node runners. Anyone who is running an nwaku node is invited to post a photo on X, tagging @waku_org and using #ProofOfNode. Waku will share your post and give you a shoutout. He also announced that they were working on early-stage node operator and onboarding programs that will be formally announced in the coming weeks. Currently, the Wak team is collecting feedback from operators on their experience setting up a node; [drop a message on Discord](https://discord.com/invite/gMPAzmcDER) to leave any feedback you may have. 
```

> ### Does the current js waku support React Native?

Reviewer: ```suggestion
### Does js-waku currently support React Native?
```

Libp2p, the modular network stack and protocol suite that Waku is built on top of, recently added support for React native. However, while it is technically enabled, the team has not been able to test it on Waku yet and as so Waku does not officially support it. Testing is on the roadmap for 2024. For now, we provide C bindings which should make it possible for mobile developers to use Waku. Find what you need in the [Guides](https://docs.waku.org/) section of the documentation site. 

### What encryption methods are currently available in the Waku library? 

> The team strives to keep Waku as unopinionated as possible, so encryption and decryption of payloads are left for application layer developers to choose from. That said, the option of using Noise is offered.

Reviewer: ```suggestion
The team strives to keep Waku as unopinionated as possible, so encryption and decryption of payloads are left for application layer developers to choose from. That said, AES, ECIES and Noise encryption are provided by the Waku protocols and libraries.
```
